app/qml/components/MMGpsDataDrawer.qml

```diff
@@ -0,0 +1,310 @@
+              descriptionText: {
+                if ( !__positionKit.hasPosition || __positionKit.horizontalAccuracy < 0 ) {
+                  return qsTr( "N/A" )
+                }
+
+                __positionKit.horizontalAccuracy.toFixed(2) + " m"
+              }
+            }
+
+            MMGpsDataText{
+              titleText: "Vertical accuracy"
+              descriptionText: {
+                if ( !__positionKit.hasPosition || __positionKit.verticalAccuracy < 0 ) {
+                  return qsTr( "N/A" )
+                }
+
+                __positionKit.verticalAccuracy.toFixed(2) + " m"
+              }
+              alignmentRight: true
+            }
+          }
+
+          MMSpacer {}
+
+          Row {
+            width: parent.width
+            height: rowHeight
+
+            MMGpsDataText{
+              titleText: "Altitude"
+              descriptionText: {
+                if ( !__positionKit.hasPosition || Number.isNaN( __positionKit.altitude ) ) {
+                  return qsTr( "N/A" )
+                }
+                __positionKit.altitude.toString() + " m"
+              }
+            }
+
+            MMGpsDataText{
+              titleText: "Satellites (in use/view)"
+              descriptionText: {
+                if ( __positionKit.satellitesUsed < 0 || __positionKit.satellitesVisible < 0 )
+                {
+                  return qsTr( "N/A" )
+                }
+
+                __positionKit.satellitesUsed + "/" + __positionKit.satellitesVisible
+              }
+              alignmentRight: true
+            }
+          }
+
+          MMSpacer {}
+
+          Row {
+            width: parent.width
+            height: rowHeight
+
+            MMGpsDataText{
+              titleText: "Speed"
+              descriptionText: {
+                if ( !__positionKit.hasPosition || __positionKit.speed < 0 ) {
+                  return qsTr( "N/A" )
+                }
+
+                __positionKit.speed.toString(2) + " km/h"
+              }
+            }
+
+            MMGpsDataText{
+              titleText: "Last Fix"
+              descriptionText: __positionKit.lastRead || qsTr( "N/A" )
+              alignmentRight: true
+            }
+          }
+
+          MMSpacer {}
+
+          Row {
+            width: parent.width
+            height: rowHeight
+
+            MMGpsDataText{
+              titleText: "GPS antenna height"
+              descriptionText: __positionKit.gpsAntennaHeight > 0 ? __positionKit.gpsAntennaHeight.toString(3) + " m" : qsTr( "Not set" )
+            }
+          }
+
+          Item {
+            width: 1
+            height: 20 * __dp
+          }
+
+          MMButton {
+            id: primaryButton
+
+            width: parent.width - 2 * 20 * __dp
+            anchors.horizontalCenter: parent.horizontalCenter
+
+            text: qsTr("Manage GPS receivers")
+
+            onClicked: {
+              console.log("GPS data drawer button test OK")
```

Comment:
this must emit signal `manageGPSClicked()`. `console.log()` should be called in gallery page